README.md

```diff
@@ -149,10 +149,31 @@ regex.
 
 ### Releasing
 
-On a new branch, run `npm version <major|minor|patch|...> -m 'rel: %s'` (see
-`npm help version` for details). It will update the version in `package.json`,
-commit the changes, and create a git tag. Now you push the branch, get it
-reviewed and merged.
+On a new branch, run
+
+```
+# tsickle releases are all minor releases for now, see npm help version.
+$ npm version minor -m 'rel: %s'
+```
+
+It will update the version in `package.json`, commit the changes, and
+create a git tag.
+
+Push the branch and get it reviewed, but *do not merge*.  If you click
+the "rebase and merge" button in the Github UI it changes the commit,
+so the git tag that was created would point at the wrong commit.
+
+Instead, push the branch to master directly via:
+
+```
+$ git push origin mybranch:master
+$ git push orgiin v0.32.0
+```
+
+Note that Github will block non-fast-forward pushes to master, so if
+there have been other intervening commits you'll need to recreate the
+release.  Once the versioned tag is pushed to Github the release will
```

Comment:
We really need to hook up mdformat here to get rid of your evil double-plenking ways... ;-)